service/discovery/discovery.go

```diff
@@ -68,6 +67,11 @@ const (
 	ProviderAzure     = "azure"
 	ProviderOpenstack = "openstack"
 	ProviderCSAF      = "csaf"
+
+	// DiscovererStart is emitted at the start of a discovery run.
+	DiscovererStart DiscoveryEventType = iota
```

Comment:
Not so sure what this is for. But it was there before, it got just moved to this const block

Reply:
That are event types for the StartDiscovery method. See lines 408 ff and 424 ff.

I moved it, because we had several const blocks.